src/Workspaces/VisualBasic/Portable/FindSymbols/VisualBasicDeclaredSymbolInfoFactoryService.vb

```diff
@@ -88,16 +87,63 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.FindSymbols
             End If
         End Sub
 
-        Private Function GetTypeName(typeSyntax As TypeSyntax) As String
+        Private Function GetTypeName(typeSyntax As TypeSyntax, Optional onlyHandleNameSyntax As Boolean = True) As String
             If TypeOf typeSyntax Is SimpleNameSyntax Then
                 Return GetSimpleName(DirectCast(typeSyntax, SimpleNameSyntax))
             ElseIf TypeOf typeSyntax Is QualifiedNameSyntax Then
                 Return GetSimpleName(DirectCast(typeSyntax, QualifiedNameSyntax).Right)
             End If
 
+            If Not onlyHandleNameSyntax Then
+                If TypeOf typeSyntax Is NullableTypeSyntax Then
+                    Return GetTypeName(DirectCast(typeSyntax, NullableTypeSyntax).ElementType)
```

Comment:
this doesn't seem right at all.  why is this going on?

Reply:
Changed since this comment.